### PR TITLE
SRCH-2593 updating translation guidance

### PR DESCRIPTION
**Pull request by amanifarooque**
## Summary
- This PR updates the CONTRIBUTING.md file to point users to search-gov, and removes the relevant information from the readme.
- This will need to go out in conjunction with SRCH-649, moving localizations from Punchcard to search-gov
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:

#### Functionality Checks

- [x] Code is functional.

- [x] Automated checks pass, if applicable. If Code Climate checks do not pass, explain reason for failures:

- [x] If your changes will be tested manually, you have run `bundle update` and committed your changes to Gemfile.lock - N/A
 
- [x] You have merged the latest changes from the target branch (usually `master` or `main`) into your branch.
 
- [x] If your target branch is NOT `master` or `main`, specify the reason:
 
- [X] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release #.#.#** matching the release number
 
- [x] You have squashed your commits into a single commit (exception: your PR includes commits with formatting-only changes, such as required by Rubocop)

- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket
 
#### Process Checks

- [x] You have specified an "Assignee", and if necessary, additional reviewers

### CODE REVIEW

**Review by MothOnMars — CHANGES_REQUESTED** [review]
a couple of copy edits

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -61,7 +45,7 @@ git push origin my-feature-branch
 
 #### Make a Pull Request
 
-Go to https://github.com/contributor/punchcard and select your feature branch. Click the 'Pull Request' button and fill out the form. A series of automated tests will run on [CirclCI](https://circleci.com/gh/GSA/punchcard). If tests pass, your pull request will usually get reviewed and accepted within a few days.
+[Create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) to propose your changes. A series of automated tests will run on [CircleCI](https://circleci.com/gh/GSA/punchcard). If tests pass, your pull request will usually get reviewed and accepted within a few days.
 
 #### Be Patient
```
[the remark this answers is not in the document]
Comment (amanifarooque) [reply CONTRIBUTING.md:50]: I'm in favor of removing that "Be Patient" section entirely, I don't think it adds anything.

[the remark this answers is not in the document]
Comment (MothOnMars) [reply CONTRIBUTING.md:50]: Works for me!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,31 +1,15 @@
 Contributing to Punchcard
 =========================
 
-Punchard is work of [multiple contributors](https://github.com/GSA/punchcard/graphs/contributors). You're encouraged to submit [pull requests](https://github.com/GSA/punchcard/pulls), [propose features and discuss issues](https://github.com/GSA/punchcard/issues).
+Punchcard is work of [multiple contributors](https://github.com/GSA/punchcard/graphs/contributors). You're encouraged to submit [pull requests](https://github.com/GSA/punchcard/pulls), [propose features and discuss issues](https://github.com/GSA/punchcard/issues).
```
[the remark this answers is not in the document]
Comment (amanifarooque) [reply CONTRIBUTING.md]: Updated!

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amanifarooque) [reply CONTRIBUTING.md]: Removed!

**Review by MothOnMars — APPROVED** [review]
LGTM

**Comment by amanifarooque** [comment]
Removing Dawn as a reviewer as we've gotten her feedback via Slack.

**Pull request merged**